### ISSUE TITLE
[Not an issue] some comments

### ISSUE THREAD

**Issue by dhk-room101** [gildor2/UEViewer#216]
I wanted to leave a comment on your forum, but for some reason I cannot register, although I tried several times in the past few days. First when attempting to register it said: internal forum error. After a few days, I finally registered, but several attempts later still didn't get the registration email, and of course I checked my junk mail :-)

Anyways, I just wanted to comment about UE5. You may recall my ueviewer fork that handles porting maps. I successfully automatically ported cooked maps from Borderlands 3 to Unreal Engine 5. It should work between any UE4 versions and current UE5. The process is the same as before because of the current high cross compatibility between UE4 and UE5. 

[https://www.youtube.com/watch?v=knywf9SobpQ](test) 

Nothing for you to do about this, I'll close this issue after submitting.

**Comment by gildor2** [comment]
"Internal forum error" usually displayed when your IP or email is blacklisted on stopforumspam service.

**Comment by dhk-room101** [comment]
I still cannot create an account, no error, I just don't get any registration email with any email address I try. anyways, I just wanted to pick your brains about an oodle 64 error 
`OodleLZ_Decompress returned 0`
I am using oo2core_5_win64.dll (size 913408 bytes)
I am running some extra tests for Jedi Fallen Order. I'm wondering maybe I'm using the wrong DLL...

**Comment by gildor2** [comment]
What asset gives you this error? I have the game installed, so I can check it myself - to see if it's some logical error in UModel, corrupted asset, or something else.

**Comment by gildor2** [comment]
Current binary build is using built-in oodle library, i.e. it is statically compiled. May be there's a problem with DLL itself, if MY build works for you, but custom build - not?

**Comment by dhk-room101** [comment]
Perhaps I can also statically link oodle, If you give me some instructions? You can also download my fork (doesn't have the DLL though, I downloaded the DLL from some DLL website) 
my fork is manually synced to 1558 build, plus my changes for umap

**Comment by gildor2** [comment]
The instructions are in libs/oodle/README.md

**Comment by dhk-room101** [comment]
Excellent, I just finished downloading 4.27, and follow the instructions in the README, and everything works as expected! 

Thanks!

Stay tune for some UE5 results :-) in particular World Partitions. I already tested Fallout 4 Commonwealth map porting to UE5, and seamlessly streamed hundreds of thousands of objects via UE5 World Partitions. Fallen order doesn't use landscape that much, so I don't have to worry about re-creating the landscape from stitching together the heightmap pieces, like I had to do for Borderlands 3
